src/common/models/Projection.ts

```diff
@@ -181,10 +180,7 @@ export class Projection {
       actualTimeseries,
       metricsTimeseries,
       dates,
-    } = this.getAlignedTimeseriesAndDates(
-      summaryWithTimeseries,
-      PROJECTIONS_TRUNCATION_DAYS,
```

Comment:
Do we still need `PROJECTIONS_TRUNCATION_DAYS` constant since futureDaysToInclude has been removed? I do not see it referenced elsewhere.

Reply:
Nope, remove it!